scripts/check_IGSR_ftp.pl

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Data::Dumper;
+use Net::FTP;
+
+## Check the content of a list of directories against their manifest files
+## Looks at FTP dir: vol1/ftp/data_collections/HGSVC3/working/;
+
+die "\nError a file of ftp directories to check is required\n\n" unless defined  $ARGV[0];
+
+
+my $ftp_dir = "vol1/ftp/data_collections/HGSVC3/working";
+my $ftp = Net::FTP->new("ftp.1000genomes.ebi.ac.uk", Debug => 0)|| die "Cannot connect to ftp.1000genomes.ebi.ac.uk: $@";    
+$ftp->login("anonymous",'-anonymous@') || die "Cannot login ", $ftp->message; 
+
+open my $dir_list, $ARGV[0] || die "Failed to open dir list $ARGV[0] : $!\n";
+
+# list format - directories to check manifest v content 
+# 2021_Hi-C_JAX/	 
+# 2021_RNAseq_JAX/
+# 20211207_UMIGS_HiFi/GM19434/
+
+while(<$dir_list>){
+  chomp;
+  my $dir = (split)[0];
+  $dir =~ s/\/$//;
+  print "Checking $dir\n";
+  download($dir);
+}
+
+$ftp->quit;
+
+sub download{
+
+  my $dir = shift;
+  my %get;
+
+  # back to top to get the right path
+  $ftp->cwd() || die "Cannot change to root dir " , $ftp->message;
+  $ftp->cwd("$ftp_dir/$dir")  || die "Cannot change project directory to $ftp_dir/$dir ", $ftp->message;
+
+  ## get list of data files per dir
+  my $data_file_list = $ftp->dir();
```

Comment:
This does not get a recursive list. Do we want to list the files recursively?
Sometimes the manifest file may not be in the root directory, we may miss the manifest file in this case unless we have a convention to have the manifest file in the root directory always.
Ex: 20220711_Strandseq